bookingticketui/src/components/ticket-form/ticket-form.jsx

```diff
@@ -84,60 +87,73 @@ const Ticket = () => {
-                      new Date(data.result?.data?.tickets[0].departureTime),
-                      "dd/MMMM/yyyy"
-                    )}
-                  </ListGroup.Item>
-
-                  <ListGroup.Item>
-                    Seat Type: {data.result?.data?.tickets[0].seatClass}
-                  </ListGroup.Item>
-
-                  <ListGroup.Item>
-                    Depart location:{" "}
-                    {data.result?.data?.tickets[0].locationFrom}
-                  </ListGroup.Item>
-
-                  <ListGroup.Item>
-                    Arrival location: {data.result?.data?.tickets[0].locationTo}
-                  </ListGroup.Item>
-                </ListGroup>
-
-                <Card.Body>
-                  <Card.Link href="#">Card Link</Card.Link>
-                  <Card.Link href="#">Another Link</Card.Link>
-                </Card.Body>
-              </Card>
+              <div
+                style={{
+                  display: "flex",
+                  gap: "10px",
+                  flexWrap: "wrap",
+                  justifyContent: "center",
+                }}
+              >
+                {tickets.map((getcon) => (
+                  <Card key={getcon.passengerName} style={{ width: "25rem" }}>
+                    <div className="Ticket-Person">
+                      <Card.Img variant="top" src="src\assets\top-icon.png" />
+
+                      <Card.Body>
+                        <Card.Title>
+                          Ticket reference code: {conForm.Code}
+                        </Card.Title>
+                      </Card.Body>
+
+                      <ListGroup className="list-group-flush">
+                        <ListGroup.Item>
+                          Name: {getcon.passengerName}
+                        </ListGroup.Item>
+
+                        <ListGroup.Item>
+                          Paid: {data.result?.data?.isPaid ? "Paid" : "Not yet"}
+                        </ListGroup.Item>
+
+                        <ListGroup.Item>
+                          Number of people: {data.result?.data?.numberPeople}
+                        </ListGroup.Item>
+
+                        <ListGroup.Item>
+                          Type of flight:
+                          {data.result?.data?.isRoundFlight
+                            ? "Round-trip"
+                            : "One-way"}
+                        </ListGroup.Item>
+
+                        <ListGroup.Item>
+                          Boarding time:{" "}
+                          {format(
+                            new Date(
+                              data.result?.data?.tickets[0].departureTime
+                            ),
+                            "dd/MMMM/yyyy"
+                          )}
+                        </ListGroup.Item>
+
+                        <ListGroup.Item>
+                          Seat Type: {data.result?.data?.tickets[0].seatClass}
+                        </ListGroup.Item>
```

Comment:
ticket[index] is not necessary here when you already have "map". Take advantage of it